mantidproject/mantid
# Move developer site build away from building on central Jenkins node

**Description of work.**

Adds a new script and Jenkinsfile to build and publish the developer site. This removes the old script that had to be built on the main Jenkins node as it required some packages to be installed first. The new script simply uses conda in an aim to unify everything around it. The Jenkinsfile was added to avoid reconfiguring a new job in the UI.

**To test:**

The new job is at [build_and_publish_developer_site](https://builds.mantidproject.org/job/build_and_publish_developer_site/). Check is has run and check the [developer repository](https://github.com/mantidproject/developer) has had a recent push.

*There is no associated issue.*





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
